product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -38,17 +48,23 @@ WHERE sub_name = 'bdr_bdrdb_bdrgroup_node2_node1';
 SELECT bdr.alter_subscription_enable ('bdr_bdrdb_bdrgroup_node2_node1');
 ```
 
-### When to use Parallel Apply
+### When to use Parallel apply
+
+Parallel apply is always on by default and for most operations, we recommend that it is left on.
+
+### When not to use Parallel apply
+
+From PGD 5.2, Parallel apply works with Group Commit, CAMO and eager replication. 
```

Comment:
```suggestion
From PGD 5.2, Parallel apply works with Group Commit, CAMO, and eager replication. 
```
Seems odd to capitalize Group Commit and not eager replication.  Again, I would think you would capitalize both or neither.

How does this text tell you when **not** to use Parallel apply?

Reply:
Section exists in the previous version of the page already in place. It's also there in the expectation that at some point soon, we are going to find circumstances when PA isn't recommended. As this is still a draft, I've been holding it open.